client/components/token-field/index.jsx

```diff
@@ -463,16 +474,24 @@ var TokenField = React.createClass( {
 		} );
 	},
 
-	_addNewToken: function( token ) {
-		var newValue;
+	_addNewToken: function( token, { isBatchOperation = false } = {} ) {
```

Comment:
I like this pattern, it's a clear way of passing an optional argument with a name.